### PR TITLE
(DOCSP-19162): Refactor Intro for Mobile Devs for Readability

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Edit the mobile developer introduction page for readability. 

To assess readability used the tools:  
- https://hemingwayapp.com/
- http://readable.com/

Refactored so that it achieved score on Readable of 'B', per [direction in the ticket's epic](https://jira.mongodb.org/browse/DOCSP-19160?focusedCommentId=4175080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4175080).

### Jira

- https://jira.mongodb.org/browse/DOCSP-19162

### Staged Changes (Requires MongoDB Corp SSO)

- [Introduction to MongoDB Realm for Mobile Developers](https://docs-mongodborg-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-19162/get-started/introduction-mobile)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/get-started/introduction-mobile.txt [hunk 1]
```diff
@@ -10,11 +10,11 @@ Introduction to MongoDB Realm for Mobile Developers
    :depth: 2
    :class: singlecol
 
-This introduction explains the {+service+} product at a high level for
-developers who primarily develop mobile applications. After
-reading, you should understand what {+service+} is, how it interacts
-with {+client-database+}, and which benefits {+backend+} offers compared
-to traditional mobile development stacks.
+This introduction explains the {+backend+} product for mobile application developers. After reading, you should understand:
```
Comment (cbush) [inline source/get-started/introduction-mobile.txt:13]: Why did you change `service` to `backend`? It should still be `service` in this case, no?

File: source/get-started/introduction-mobile.txt [hunk 2]
```diff
@@ -112,44 +90,30 @@ easy to share data across users and platforms.
...
 
-{+service+} enables sync between {+client-database+} and {+atlas+},
-seamlessly stitching together the two components into an application
-layer for your mobile app. To support these services, {+backend+} also
-provides components to fulfill several common application requirements:
+{+backend+} also provides services to fulfill several common application backend requirements:
 
 - :ref:`{+service-short+} Users and Authentication <users-and-authentication>`
-  provide built-in user management with easy integration into
+  provide built-in user management. It's easy to integrate with
```
Comment (cbush) [inline source/get-started/introduction-mobile.txt:96]: I would avoid saying anything is easy (in general as this can be alienating)
```suggestion
  provide built-in user management. Integrate with
```

File: source/get-started/introduction-mobile.txt [hunk 3]
```diff
@@ -169,27 +133,20 @@ Summary
 -------
 
 - {+service+} is a serverless application platform that takes care of
-  the details of deployment and scaling for you. You can customize your
-  {+app+} with custom logic via functions and triggers, custom
-  permissions via rules, and authentication via third-party SSO.
-
+  the details of deployment and scaling for you. 
+- You can build your {+app+} with functions and triggers, custom
```
Comment (cbush) [inline source/get-started/introduction-mobile.txt:137]: I wouldn't say "build" here as the app specifically refers to the stuff on the Realm UI. Making this its own bullet actually makes it feel out of place on this page.
```suggestion
- You can customize your {+app+} with functions and triggers, custom
```

**Review by cbush — APPROVED** [review]

File: source/get-started/introduction-mobile.txt (hunk 1 above)
Comment (mongodben) [reply source/get-started/introduction-mobile.txt:13]: that wasn't an intentional change. i copied & pasted the refactored versions in, and then manually added the source constants back. so this must have just been a mistake in that migration.
